src/android/OpenTokAndroidPlugin.java

```diff
@@ -741,26 +836,72 @@ public void onSignalReceived(Session arg0, String arg1, String arg2, Connection
         }
     }
 
+    // archiveListener
+    public void onArchiveStarted(Session session, String id, String name) {
+        JSONObject data = new JSONObject();
+        try {
+            data.put("id", id);
+            data.put("name", name);
+            triggerJSEvent("sessionEvents", "archiveStarted", data);
+        } catch (JSONException e) {
+        }
```

Comment:
in case this fails, can log it in the java code that an archive was started with the following id and name?